templates/standards/nist-csf.json

```diff
@@ -300,8 +338,8 @@
           "summary": "Access to systems and assets is controlled, incorporating the principle of least functionality."
         },
         {
-          "ref": "PR.PT-4",
```

Comment:
What happened to this?

Reply:
I'm not sure how I fat-fingered it. I made the update/fix